iocore/net/quic/QUICConfig.cc

```diff
@@ -82,8 +82,13 @@ quic_init_client_ssl_ctx(const QUICConfigParams *params)
     SSL_CTX_sess_set_new_cb(ssl_ctx.get(), QUIC::ssl_client_new_session);
   }
 
+  // TODO: QUIC-specific keylogging is deprecated in favor of the more broad
```

Comment:
Although the removal doesn't have to be made on this PR, I don't think it have to wait for a future release. There is a disclaimer below on the docs. At least I don't mind removing it immediately.

> All configurations for QUIC are still experimental and may be changed or
removed in the future without prior notice.